install/boards/bcm_27xx.sh

```diff
@@ -2,11 +2,15 @@
 
 echo "Configuring BCM27XX board (Raspberry Pi 4).."
 
+VERSION="${VERSION:-master}"
+REMOTE="${REMOTE:-https://raw.githubusercontent.com/bluerobotics/companion-docker}"
+REMOTE="$REMOTE/$VERSION"
```

Comment:
this overwrites the environment variable
Try ROOT="$REMOTE/$VERSION"

Reply:
Thanks, can you check the latest version ? It should fix it

Reply:
Ok, that should be one